cms/src/components/content/EditContentBasic.vue

```diff
@@ -1,133 +1,47 @@
 <script setup lang="ts">
-import LInput from "@/components/forms/LInput.vue";
 import LCard from "@/components/common/LCard.vue";
-import { PencilIcon } from "@heroicons/vue/16/solid";
-import { PublishStatus, type ContentDto } from "luminary-shared";
-import { nextTick, ref, watch } from "vue";
-import { Slug } from "@/util/slug";
+import ContentBasic from "../content/ContentBasic.vue";
+import ContentSEO from "../content/ContentSEO.vue";
+import { ref } from "vue";
+import type { ContentDto } from "luminary-shared";
 
 type Props = {
     disabled: boolean;
 };
 defineProps<Props>();
-const content = defineModel<ContentDto>("content");
-
-// Slug generation
-const isEditingSlug = ref(false);
-const slugInput = ref<HTMLInputElement | undefined>(undefined);
-
-const startEditingSlug = () => {
-    isEditingSlug.value = true;
-    nextTick(() => {
-        slugInput.value?.focus();
-    });
-};
-
-let previousTitle: string = content.value?.title || "";
-let previousSlug: string = content.value?.slug || "";
-watch(
-    content,
-    async () => {
-        if (!content.value) return;
-
-        const titleChanged = previousTitle != content.value.title;
-        const slugChanged = previousSlug != content.value.slug;
-
-        // Only update the slug if the title or slug has changed
-        if (!titleChanged && !slugChanged) {
-            return;
-        }
-
-        // If the title is empty, generate a new slug
-        if (!content.value.title) {
-            content.value.slug = "";
-        }
 
-        // If the slug is empty, generate a new one from the title
-        if (!content.value.slug) {
-            content.value.slug = Slug.generateNonUnique(content.value.title);
-        }
-
-        // Auto-update the slug if the title changes when in draft mode (unless the slug has been manually changed)
-        if (
-            titleChanged &&
-            content.value.status == PublishStatus.Draft &&
-            content.value.slug.replace(/-[0-9]*$/g, "") ==
-                Slug.generateNonUnique(previousTitle).replace(/-[0-9]*$/g, "")
-        ) {
-            // TODO: This sometimes creates a race condition
-            content.value.slug = Slug.generateNonUnique(content.value.title);
-        }
-
-        // Validate slug
-        if (slugChanged) {
-            content.value.slug = Slug.generateNonUnique(content.value.slug);
-        }
-
-        previousTitle = content.value.title;
-        previousSlug = content.value.slug;
-    },
-    { deep: true },
-);
+const content = defineModel<ContentDto>("content");
 
-const validateSlug = async () => {
-    if (!content.value) return;
-    content.value.slug = await Slug.generate(content.value.slug, content.value._id || "");
-};
+const activeTab = ref(0);
+const tabs = [
+    { title: "General ", component: ContentBasic },
+    { title: "SEO ", component: ContentSEO },
```

Comment:
I would keep the naming with the prefix `Edit` so all edit components are grouped together. Perhaps rename this component `EditContentGeneral`, because that is the title of it?

- `EditContentGeneral`
    -`EditContentBasic`
    - `EditContentSeo` (<- please don't capitalize SEO, we follow camel case even there).